### ISSUE TITLE
[DS - Unit 4] Sample program uses plot lib with tables instead of lists

### ISSUE THREAD

**Issue by schanzer** [bootstrapworld/curr#257]
We should update this to use the new API

**Comment by SDooman** [comment]
We're still waiting on all of the functions we want to be merged into CPO.  I'll update as soon as they're pushed

**Comment by SDooman** [comment]
This is done.
